ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ota-multi-stable-4.18-upgrade-from-stable-4.17.yaml

```diff
@@ -85,6 +95,53 @@ tests:
     - ref: cucushift-upgrade-healthcheck
     - ref: cucushift-ota-postupgrade
     workflow: cucushift-installer-rehearse-azure-ipi
+- as: baremetal-ipi-ovn-ipv4-fips-vmedia-amd-f28
+  capabilities:
+  - intranet
+  cron: 36 16 8 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
+      ENABLE_OTA_TEST: OCP-25473
+      FIPS_ENABLED: "true"
+      RESERVE_BOOTSTRAP: "false"
+      architecture: amd64
+      masters: "3"
+      workers: "2"
+    test:
+    - as: set-upgrade-releases
+      commands: echo "${INTERMEDIATE_RETARGET}" > ${SHARED_DIR}/upgrade-edge
+      dependencies:
+      - env: INTERMEDIATE_RETARGET
+        name: release:intermediate1
+      from: cli
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    - chain: openshift-upgrade-qe-test
```

Comment:
Happen to find all the test cost a long time, do you intend to use chain `openshift-upgrade-qe-test` for the retarget test? The chain includes e2e test, which seems not necessary here.

Reply:
Sure, will change to openshift-upgrade-qe-sanity

Reply:
`openshift-upgrade-qe-sanity` is better than `openshift-upgrade-qe-test`. It still has several unnecessary steps, right? Combining with https://github.com/openshift/release/pull/58136#discussion_r2046150272, IMO following steps should be enough:

```
  - ref: cucushift-upgrade-retarget
  - ref: cucushift-upgrade-healthcheck
```